templates/common/root/_Gruntfile.js

```diff
@@ -184,13 +184,15 @@ module.exports = function (grunt) {
     },
 
     // Add vendor prefixed styles
-    autoprefixer: {
+    postcss: {
       options: {
-        browsers: ['last 1 version']
+        processors: [
+          require('autoprefixer-core')({browsers: ['last 1 version']})
```

Comment:
Little strange that we lately dropped suppport for IE8 (https://github.com/yeoman/generator-angular/commit/7c6f245b73ce59782aed0a1efd300e83569eb0e8), while it in fact the support was IE10 anyways.

But yeah, I agree.